cmd/datamon/cmd/context_get.go

```diff
@@ -4,3 +4,76 @@
  */
 
 package cmd
+
+import (
+	"bytes"
+	"context"
+	"log"
+
+	context2 "github.com/oneconcern/datamon/pkg/context"
+	status "github.com/oneconcern/datamon/pkg/core/status"
+	"github.com/oneconcern/datamon/pkg/model"
+
+	"github.com/oneconcern/datamon/pkg/storage/gcs"
+	"github.com/spf13/cobra"
+	"golang.org/x/sys/unix"
+)
+
+var ContextGetCommand = &cobra.Command{
+	Use:   "get",
+	Short: "Get a context info",
+	Long:  "Get a Datamon context's info",
+	Run: func(cmd *cobra.Command, args []string) {
+		getContext()
+	},
+}
+
+func getContext() {
+	configStore, err := gcs.New(context.Background(), datamonFlags.core.Config, config.Credential)
+	if err != nil {
+		wrapFatalln("failed to create config store. ", err)
+	}
+	contextName := datamonFlags.context.Descriptor.Name
+	has, err := configStore.Has(context.Background(),
+		model.GetPathToContext(contextName))
+	if err != nil {
+		wrapFatalln("failed to test if context exists. ", err)
```

Comment:
Since we have introduced non-fatal mocks, we should make sure test runs follow the same path:
```suggestion
		wrapFatalln("failed to test if context exists. ", err)
		return
```